File: week-6/morning-practice/student.py
class Student:
    def __init__(self):
        self.grade = []

    def add_grade(self, grade):
        self.grade.append(grade)

    def get_average(self):
        return round(sum(self.grade) / len(self.grade), 2)



akos = Student()
akos.add_grade(5)
akos.add_grade(4)
akos.add_grade(4)

print(akos.get_average())

File: week-7/js-workshop-wed/student.js
'use strict';

function Student(name) {
  this.name = name;
  this.grades = [];
  this.addGrade = function(grade) {
    this.grades.push(grade);
  }
  this.getAverage = function () {
      return this.grades.reduce(function (acc, num) {
        return acc + num;
      }, 0) / this.grades.length;
  }
}

var jozsi = new Student('Jozsef');
jozsi.addGrade(4);
jozsi.addGrade(3);
jozsi.addGrade(5);
jozsi.addGrade(5);

console.log(jozsi.getAverage());
